cfpb/cfgov-refresh
# Ask CFPB answer pages

Updates English and adds Spanish templates for Ask CFPB answer pages

## Additions

- Spanish templates
- Spanish feedback form styling/handling/messaging

## Changes

- Update English Ask CFPB template content

## Testing

- [Migrate knowledgebase](https://github.com/cfpb/cfgov-refresh/pull/2845) if necessary
- Run gulp
- Navigate to [English](http://localhost:8000/ask-cfpb/are-there-any-different-requirements-for-same-sex-spouses-jointly-applying-for-a-va-loan-en-2061/) and [Spanish](http://localhost:8000/obtener-respuestas/existen-requisitos-diferentes-para-conyuges-del-mismo-sexo-cuando-solicitan-un-prestamo-de-va-de-manera-conjunta-es-2061/) answer pages and verify their content/feedback functionality
- Verify that forms still work correctly on other pages like http://localhost:8000/owning-a-home/ and http://localhost:8000/owning-a-home/help-us-improve

## Screenshots

<img width="1321" alt="screen shot 2017-04-14 at 9 53 42 am" src="https://cloud.githubusercontent.com/assets/778171/25053915/2fba341e-210f-11e7-9b60-53aedd474c9d.png">

<img width="1317" alt="screen shot 2017-04-14 at 9 52 48 am" src="https://cloud.githubusercontent.com/assets/778171/25053917/33e90bbe-210f-11e7-868f-bceda883bc8d.png">



## Notes

- The Spanish printable template has been imported but isn't hooked up yet
- I added some basic messaging/form handling for the Spanish feedback form, but it's pretty ad hoc. We'll probably need a more cohesive approach if we bring in more Spanish functionality


## Checklist
